### ISSUE TITLE
Problem converting Siemens rda

### ISSUE THREAD

**Issue by mdefende** [wtclarke/spec2nii#46]
Hi,

I'm new to MRS data and am trying to convert some rda files but am running into an error. Here is the traceback:

```
Traceback (most recent call last):
  File "/home/mdefende/.conda/envs/fslmrs/bin/spec2nii", line 10, in <module>
    sys.exit(main())
  File "/home/mdefende/.conda/envs/fslmrs/lib/python3.10/site-packages/spec2nii/spec2nii.py", line 521, in main
    spec2nii(*args)
  File "/home/mdefende/.conda/envs/fslmrs/lib/python3.10/site-packages/spec2nii/spec2nii.py", line 272, in __init__
    args.func(args)
  File "/home/mdefende/.conda/envs/fslmrs/lib/python3.10/site-packages/spec2nii/spec2nii.py", line 388, in rda
    self.imageOut, self.fileoutNames = convert_rda(args.file, args.fileout, args.verbose)
  File "/home/mdefende/.conda/envs/fslmrs/lib/python3.10/site-packages/spec2nii/rda.py", line 36, in convert_rda
    if len(match.groups()) < 2:
AttributeError: 'NoneType' object has no attribute 'groups'
```

The PI says it's a standard MRS sequence and isn't sure why there would be an issue with it compared with others. I was just hoping for some advice on this. Thanks

**Comment by mdefende** [comment]
I was using the base command for conversion `spec2nii rda <rda_path>`

**Comment by wtclarke** [comment]
Hi @mdefende 

I've done less work on the rda conversion side, having not used it much myself. Could you provide a small sample of data?

**Comment by mdefende** [comment]
@wtclarke Thank you for the response. The PI has sent over some pilot data you can find on [google drive](https://drive.google.com/drive/folders/1oxIdruvtKvo0omFI9UGutvBpHelHu-5C?usp=sharing). The metabolite file is labelled WS and the reference image is labelled NWS. Hopefully this will be sufficient, but let me know if you need anything else for this. Thanks!

**Comment by wtclarke** [comment]
Hi @mdefende, there was a minor bug in some regex. This should be fixed in v0.4.7 to be published shortly. Thanks for providing the data.
